pkg/provider/okta/okta.go

```diff
@@ -681,6 +685,14 @@ func getMfaChallengeContext(oc *Client, mfaOption int, resp string) (*mfaChallen
 	oktaVerify := gjson.Get(resp, fmt.Sprintf("_embedded.factors.%d._links.verify.href", mfaOption)).String()
 	mfaIdentifer, _ := parseMfaIdentifer(resp, mfaOption)
 
+	if !strings.Contains(oktaVerify, "rememberDevice") {
+		separator := "?"
+		if strings.Contains(oktaVerify, "?") {
```

Comment:
Why don't you use the separator variable defined above here?

Reply:
The intent here is to use `?` to add query parameter if `oktaVerify` doesn't have any query parameters yet, and to use `&` to add the new query parameter if `oktaVerify` already has some of its own.